roles/zuul/files/jobs/hoist.yaml

```diff
@@ -22,3 +23,4 @@
       - shell: |
           ./tests/signed-off-by-test.sh
           python tests/layout-checks.py job-list.txt
+          ./tests/shellcheck-test.sh
```

Comment:
They happen in parallel iirc.

https://blog.travis-ci.com/2012-11-28-speeding-up-your-tests-by-parallelizing-them/